bulbs/api/views.py

```diff
@@ -509,7 +509,11 @@ def list(self, request):
                 active = get_query_params(self.request).get('active', '').lower()
                 now = timezone.now()
                 if active == 'true':
-                    qs = qs.filter(start_date__lte=now, end_date__gte=now)
+                    qs = qs.filter(
+                        start_date__lte=now, end_date__gte=now
+                    ) | qs.filter(
```

Comment:
@mparent61 decided to go with the union of the two querysets when updating all of these filters